### DISCUSSION TITLE
How to tell time on marble diagrams?

### DISCUSSION THREAD

**Discussion by ghost** [ReactiveX/rxjs#7031] [Report issues other than bug]
I'm reading the docs to see how RxJS de-bounces/throttles/audits observables, but there is no way to tell how much time has passed on the diagrams. I'd suggest adding tick marks at certain intervals to tell time, just like a ruler has tick marks to represent length.

**Comment by kwonoj** [comment]
It is deliberate design decision to testscheduler's frame does not represent time. So 1 frame advance does not mean any actual time passage or correlations you can make. Having a tick mark probably would be a separate enhancement, but probably that'll be handled by some 3rd party test framework I'd say.

**Reply by ghost** [reply]
Ok. Thank you.
